src/sonic-py-common/sonic_py_common/syslogger.py

```diff
@@ -0,0 +1,62 @@
+import os
+import sys
+import logging
+from logging.handlers import SysLogHandler
+
+
+class SysLogger:
+    """
+    SysLogger class for Python applications using SysLogHandler
+    """
+
+    DEFAULT_LOG_FACILITY = SysLogHandler.LOG_USER
+    DEFAULT_LOG_LEVEL = SysLogHandler.LOG_INFO
```

Comment:
should default logger level be NOTICE? The old Logger class in logger.py is default to NOTICE level.